network/containerizer/bridgepolicy.go

```diff
@@ -393,6 +367,25 @@ func (p *BridgePolicy) determineContainerSpaces(
 	return spaces.Values(), nil
 }
 
+// spaceNamesForPrinting return a sorted, comma delimited, string containing
+// the space names for each of the given space ids.
+func (p *BridgePolicy) spaceNamesForPrinting(ids set.Strings) string {
```

Comment:
Here again, if we had a single member of type `SpaceInfos`, we could implement a filter on that type and just lean on its `String` method.